src/Compiler/Compiler.csproj

```diff
@@ -5,6 +5,7 @@
     <Description>Infer.NET is a framework for running Bayesian inference in graphical models. It can also be used for probabilistic programming. This package contains the Infer.NET Compiler, which takes model descriptions written using the Infer.NET API and converts them into inference code.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyName>$(AssemblyNamePrefix)Microsoft.ML.Probabilistic.Compiler</AssemblyName>
+    <PackageId>$(AssemblyName)(NuGetPackageIdSuffix)</PackageId>
```

Comment:
Lacks `$`.

Did this actually produce the correct packages?

What will `$(NugetPackageIdSuffix)` be set to? Could you please provide examples of how the package names would look like?

Reply:
Discussed in person.